target/linux/imx/image/common.mk

```diff
@@ -49,3 +49,13 @@ define Build/imx-sdcard
 
 	$(Build/imx-combined-image-clean)
 endef
+
+define Build/imx-sdcard-raw-uboot
```

Comment:
@Leo-PL isn't the default configuration for SPL for this board capable of starting U-Boot from e.g. ext filesystem? This way only the SPL is burn in a raw mode on eMMC and rest is easily accessible in `boot` partition.

Reply:
I tried that and it didn't work, but I may have written incorrect artifact there. I presume that I just need to put u-boot.bin on /boot and that's it, like it is done already for Apalis eMMC?

Reply:
Not sure about Apalis but during my tests of UDOO Neo board (SD, not eMMC), SPL was capable of booting `uboot-dtb.img` from the `/boot` partition and this recipe worked just fine: `IMAGE/combined.bin := append-rootfs | pad-extra 128k | imx6-sdcard`

Reply:
Have a look at a0528cab445ad6bb4b372d4484bac10f638c433f

Reply:
Tried again and didn't work, despite I've seen both `u-boot-dtb.img` and `u-boot.img` in include/configs/imx7_spl.h in CONFIG_SPL_FS_LOAD_PAYLOAD_NAME - because neither CONFIG_SPL_FS_FAT nor CONFIG_SPL_FS_EXT4 is set for that board in upstream U-boot. So I guess this image should stay.